src/Security/CMSSecurity.php

```diff
@@ -34,19 +37,39 @@ class CMSSecurity extends Security
      */
     private static $reauth_enabled = true;
 
+    /**
+     * @var string Default CMS Authenticator
+     */
+    protected static $default_authenticator = 'cms';
```

Comment:
Because of the fallback method in Security::login(), the default_authenticator is back.